code/FinanceManager.Components/wwwroot/css/time-series-card.css

```
@@ -0,0 +1,140 @@
+/* Header value placeholder shown while the series loads. */
+.fm-tsvc-shimmer {
+    width: 160px;
+    height: 30px;
+    border-radius: 4px;
+    flex: none;
+    background: linear-gradient(90deg, rgba(128, 128, 128, 0.12) 25%, rgba(128, 128, 128, 0.22) 37%, rgba(128, 128, 128, 0.12) 63%);
+    background-size: 400% 100%;
+    animation: fmTsvcShimmer 1.4s ease infinite;
+}
+
+@keyframes fmTsvcShimmer {
```

Stylelint (17.12.0)

[error] Expected keyframe name "fmTsvcShimmer" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)